### PR TITLE
#5: turn off pretty for deploy

### PR DESCRIPTION
**Pull request by mithun-daa**
Let me know if this is the right approach or if you had something else in mind.

### CODE REVIEW

File: build-tasks/markup.js [hunk 1]
```diff
@@ -15,7 +20,7 @@ var gulp      = require('gulp'),
   compile = function() {
     return gulp.src(src.docs)
       .pipe(plugins.plumber())
-      .pipe(env.deploy ? plugins.pug(): plugins.pug(opts.pug))
+      .pipe(turnOffPrettyPrint())
```
Comment (jh3y) [inline build-tasks/markup.js]: Personally, I would say a function would be a little overkill for this. Also, the naming wouldn't be right. `turnOffPrettyPrint` would imply that the function `always` turns off pretty printing but it doesn't. It's only a small check that is only specific to if you decide to use `pug` for markup compilation, so a one-liner will do as most people are likely to scrap the markup compilation anyhow.

Just the line
```js
if (env.deploy && opts.pug.pretty) opts.pug.pretty = false;
```
at the top of the `compile` function will do 👍

**Comment by mithun-daa** [comment]
I have made the changes you suggested. Let me know if it does not look right. 
Thank you.

File: build-tasks/markup.js [hunk 2]
```diff
@@ -13,9 +13,11 @@ var gulp      = require('gulp'),
   },
   /* markup:compile */
   compile = function() {
+    if (env.deploy && opts.pug.pretty) opts.pug.pretty = false;
+
     return gulp.src(src.docs)
       .pipe(plugins.plumber())
-      .pipe(env.deploy ? plugins.pug(): plugins.pug(opts.pug))
+      .pipe(turnOffPrettyPrint())
```
Comment (jh3y) [inline build-tasks/markup.js]: The function `turnOffPrettyPrint` is still present.

This just needs to be `plugins.pug(opts.pug)` 👍

**Comment by mithun-daa** [comment]
😞 Sorry. Fixed it. Thank you for hand-holding me through the entire process.

**Comment by mithun-daa** [comment]
Wow! this def. needs some serious patience from the maintainer. Thank you again.

File: build-tasks/markup.js [hunk 3]
```diff
@@ -13,9 +13,11 @@ var gulp      = require('gulp'),
   },
   /* markup:compile */
   compile = function() {
+    if (env.deploy && opts.pug.pretty) opts.pug.pretty = false;
+
```
Comment (jh3y) [inline build-tasks/markup.js]: Let's just remove this blank line and we're all good! 👍  We will get there np 😄

**Comment by jh3y** [comment]
🎉  👍 

We got there 😄 

Good stuff! And thanks for contributing. Be sure to share your #Hacktoberfest contribution on Twitter!

@jh3y

**Comment by mithun-daa** [comment]
Yay!! Thanks a ton!! 👏

**Pull request merged**